src/components/menu/menu.ts

```diff
@@ -113,6 +104,8 @@ export class Menu extends Component<Props, SpreadsheetChildEnv> {
   private menuRef = useRef("menu");
   private position: DOMCoordinates = useAbsoluteBoundingRect(this.menuRef);
 
+  static timeOut: any = null;
```

Comment:
any ?

Reply:
And why is it static? There's no reason for all the menu to share a timeout is there ? 

And  the timeout should be cleared when the menu is unmounted

edit: I guess you static'd it so you can cancel the timeout of a menu when we start hovering another menu ? Just cancel the timeout when the mouse leaves the menu